src/client.rs

```diff
@@ -262,29 +262,40 @@ pub fn main() -> Result<(), String> {
 
     let mut color_selection = 0;
     let mut plane_selection = 0;
+    let mut name = whoami::username();
 
     let mut event_pump = sdl.event_pump().expect("Could not get event pump");
     
+    video_subsystem.text_input().start();
+
     'mainloop: loop {
         let menu_state = &mut MenuState::new();
 
         menu_state.color_selection = color_selection;
         menu_state.plane_selection = plane_selection;
+        menu_state.name = name;
 
         'menuloop: loop {
             for event in event_pump.poll_iter() {
                 match event {
                     Event::Quit{..} => break 'mainloop,
                     Event::KeyDown {keycode: Some(kc), ..} => {
-                        if kc == Keycode::Return || kc == Keycode::Space {
-                            color_selection = menu_state.color_selection;
-                            plane_selection = menu_state.plane_selection;
-                            break 'menuloop;
+                        match kc {
+                            Keycode::Return => {
```

Comment:
Oh no, space for start game won't work anymore :(

Reply:
And it shouldn't.